code/index.html

```diff
@@ -1,30 +1,104 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+<html lang="en-US">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <title>Simple Responsive Layout</title>
+        <link rel="stylesheet" href="dist/style.css">
+        
+    </head>
+    <body>
+        
+        <!-- Main Content -->
+        <main>
+            <!-- Header -->
+            <header class="header">                            
+                <p><a href="mailto:bergman.signe@gmail.com" class="square_btn">IN NEED OF A BABYSITTER? LOOK NO FURTHER!</a></p>
+            </header>
+
+            <!-- Hero -->
+            <section class="hero">
+                <picture class="hero-image">
+                    <source media="(min-width:1200px)" srcset="./assets/desktopsmall.png">
+                    <source media="(min-width:768px)" srcset="./assets/tabletsmall.png">
+                    <img src="./assets/mobilsmall.png" alt="Signe" style="width:auto;">
+                  </picture> 
+
+                <div class="hero-title">
```

Comment:
Hmm, I don't see closing tag for this `div`

Reply:
Hm, is that why I hade so much trouble with the header title? *fixed*